conf/broker.conf

```diff
@@ -387,3 +387,9 @@ webSocketNumIoThreads=8
 
 # Number of connections per Broker in Pulsar Client used in WebSocket proxy
 webSocketConnectionsPerBroker=8
+
+
+### --- Metrics --- ###
+
+# Enable topic level metrics
+topicMetricsEnabled=true
```

Comment:
Since this is only relative to Prometheus metrics, I would name it something like 

`exposeTopicLevelMetricsInPrometheus=true`